vite.config.ts

```diff
@@ -0,0 +1,68 @@
+import { defineConfig } from "vite";
+import react from "@vitejs/plugin-react-swc";
+import { VitePWA } from "vite-plugin-pwa";
+import path from "path";
+
+export default defineConfig({
+  root: "src",
+  base: "/junior/",
```

Comment:
This is for hosting on GitHub Pages. Assets will be under `${origin}/junior` so need to account for the path here

Reply:
Should also set this as an env variable